bin/templates/scripts/cordova/lib/check_reqs.js

```diff
@@ -121,15 +121,22 @@ module.exports.check_cocoapods = function (toolChecker) {
         // check whether the cocoapods repo has been synced through `pod repo` command
         // a value of '0 repos' means it hasn't been synced
         .then(function (toolOptions) {
+            if (toolOptions.ignore) return toolOptions;
```

Comment:
a bit nit-picky, but what is your code styleguide in regards to `{ }` after an `if` statement?

Reply:
Our ESLint style is pretty liberal in this aspect, I think. My personal styleguide says "prefer return statement on same line & without braces if line does not get too long"

Reply:
I also prefer `return`, `throw`, and sometimes even variable setting with out using `{ }` as long as the line is not too long.